README.md

```diff
@@ -119,6 +119,49 @@ Connection: keep-alive
 
 This error shows the request is missing a required property `name` from the HTTP request body.
 
+### HTTP API
```

Comment:
Do we want this in the main README? It’s getting rather full.

Maybe we could have one example in the main readme then link to the package README for the rest of it? Would that be a decent compromise?

Reply:
That's a good plan @philsturgeon I will pick it up from @mallachari and make the adjustements